cmd/kubernetes-static/main.go

```diff
@@ -179,18 +180,18 @@ func main() {
 		},
 	}
 
-	ksmClient, err := ksm.NewKSMClient(log.NewStdErr(true))
+	ksmClient, err := client.NewKSMClient(log.NewStdErr(true))
```

Comment:
I would rename the import and the function to something like "ksmclient.New()"

Reply:
I saw you did it already in the real main 👍

Reply:
Done in the other main as well!